simd/src/Kokkos_SIMD_AVX2.hpp

```diff
@@ -791,6 +823,41 @@ class simd<std::int64_t, simd_abi::avx2_fixed_size<4>> {
       _mm256_add_epi64(static_cast<__m256i>(lhs), static_cast<__m256i>(rhs)));
 }
 
+// Shift right arithmetic for 64bit packed ints is not availalbe in AVX2
+// KOKKOS_IMPL_HOST_FORCEINLINE_FUNCTION
+// simd<std::int64_t, simd_abi::avx2_fixed_size<4>> operator>>(
+//     simd<std::int64_t, simd_abi::avx2_fixed_size<4>> const& lhs,
+//     int rhs) noexcept {
+//   return simd<std::int64_t, simd_abi::avx2_fixed_size<4>>(
+//       _mm256_srai_epi64(static_cast<__m256i>(lhs), rhs));
+// }
+
+// KOKKOS_IMPL_HOST_FORCEINLINE_FUNCTION
+// simd<std::int64_t, simd_abi::avx2_fixed_size<4>> operator>>(
+//     simd<std::int64_t, simd_abi::avx2_fixed_size<4>> const& lhs,
+//     simd<std::int64_t, simd_abi::avx2_fixed_size<4>> const& rhs) noexcept {
+//   return simd<std::int64_t, simd_abi::avx2_fixed_size<4>>(
+//       _mm256_srav_epi64(static_cast<__m256i>(lhs),
+//       _mm256_cvtepi32_epi64(static_cast<__m128i>(static_cast<__m128i>(rhs))));
+// }
```

Comment:
Can you create a follow up issue for this, we may want to add a manual thing here, potentially guarded by some perf warning or something.

Reply:
Yes, in fact, I think all fallback implementations in `Kokkos_SIMD_Common.hpp` should eventually be moved to their corresponding AVX2 backend impl.

Reply:
Added as another task in #5674. Let me know if opening a separate issue is preferred.

Reply:
oh so there is a fallback already in place just elsewhere? That is fine for now